subprojects/packagefiles/spdlog/meson.build

```diff
@@ -24,7 +24,9 @@ if get_option('external_fmt').enabled()
 endif
 
 if get_option('std_format').enabled()
-  spdlog_compile_args += '-DSPDLOG_USE_STD_FORMAT'
+  if meson.get_compiler('cpp').has_header('format', required: true)
```

Comment:
required can take a feature option directly.

Reply:
I'm aware that required can take `get_option('std_format'))` directly but this whole branch is gated by the  `if get_option('std_format').enabled()` anyway so the header check is always required.
If the option isn't specified in the first if statement, the `has_header()` check will never execute.
Setting required to `true` is a logical optimization.

This is what I think you're proposing, please clarify if my assumption is wrong:
```meson
if get_option('std_format').enabled()
  if meson.get_compiler('cpp').has_header('format', required: get_option('std_format'))
    spdlog_compile_args += '-DSPDLOG_USE_STD_FORMAT'
  endif
endif
```